src/lib/components/SubscribeForm/index.js

```diff
@@ -0,0 +1,243 @@
+    this.needsDoubleOptIn = [
+      "AT: Austria",
+      "DE: Germany",
+      "GR: Greece",
+      "LU: Luxembourg",
+      "NO: Norway",
+    ];
+    this.robotName = "is-it-just-me-or-was-this-form-filled-out-by-a-robot";
+    this.submissionUrl =
+      "https://services.google.com/fb/submissions/591768a1-61a6-4f16-8e3c-adf1661539da/";
+    this.submitted = false;
+  }
+
+  /**
+   * @param {FormData} form
+   * @return {FormData}
+   */
+  cleanForm(form) {
+    const country = form.get("Country");
+    this.checkboxes.forEach((box) => {
+      const optedIn = this.needsDoubleOptIn.includes(country)
+        ? "Unconfirmed"
+        : "True";
+      form.set(box, optedIn);
+    });
+
+    form.append("LanguagePreference", "en-US");
+    form.delete(this.robotName);
+
+    return form;
+  }
+
+  submit(form) {
+    return new Promise((resolve, reject) => {
+      const request = new XMLHttpRequest();
+      request.open("POST", this.submissionUrl);
+      request.send(form);
+      request.onload = function() {
+        const response = request.response ? JSON.parse(request.response) : {};
+
+        if (response && response.result === "accepted") {
+          resolve();
+        } else if (response && response.errors) {
+          reject(response.errors);
+        } else {
+          reject("Could not submit, please try again.");
+        }
+      };
+    });
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    if (this.submitted === false) {
+      const form = new FormData(e.target);
+      const formIsRobot = form.get(this.robotName).length !== 0;
+
+      if (!formIsRobot) {
+        const cleanedForm = this.cleanForm(form);
+        this.submit(cleanedForm)
+          .then(() => (this.submitted = true))
+          .catch((e) => (this.errors = e));
+      } else {
+        this.submitted = true;
+      }
+    }
+
+    return false;
+  }
+
+  renderForm() {
+    let errorMessage;
+
+    if (this.errors) {
+      if (typeof this.errors === "string") {
+        errorMessage = html`
+          <div class="web-subscribe-error">
+            <p>${this.errors}</p>
+          </div>
+        `;
+      } else {
+        const errorMessages = Object.values(this.errors).map(
+          (e) =>
+            html`
+              <p>${e}</p>
+            `,
+        );
+        errorMessage = html`
+          <div class="web-subscribe-error">
+            ${errorMessages}
+          </div>
+        `;
+      }
+    }
+
+    const countriesOptions = countries.map((country) => {
+      return country[1] === "United States"
+        ? html`
+            <option selected value="${country[0]}">${country[1]}</option>
+          `
+        : html`
+            <option value="${country[0]}">${country[1]}</option>
+          `;
+    });
+
+    return this.submitted
+      ? ""
+      : html`
+          <form @submit="${this.handleSubmit}">
+            <div class="web-subscribe-field">
+              <input
```

Comment:
Does the subscribe form require two fields for Name? Names come in many shapes and sizes.

Reply:
It's also based on the current form which asks for first/last, I'm not sure who would be the right person to ask this.

Reply:
Sure, if we're just dialing up the form we can't change this requirement.